exercises/pig-latin/src/example.c

```diff
@@ -0,0 +1,123 @@
+#include "pig_latin.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+#include <stdbool.h>
+
+static bool is_vowel(const char letter)
+{
+   return letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o'
+       || letter == 'u';
+}
+
+static bool starts_with_vowel(const char letter_a, const char letter_b)
```

Comment:
I understand your desire to keep the variable names brief, but as-is this function is quite cryptic. It's not clear to me from the naming or context that `letter_a` and `letter_b` refer to the first and second letters in a string. I actually think it would be more clear to just pass a `const char *` and use indexing to get the first two letters.

It's also quite odd to see a function called `starts_with_vowel` that has three `true` paths where only one of those is because the first letter is a vowel. Without looking at tests I'm confused as to what this function really determines.

Reply:
Completely agree.
Will give my best to make it more sane.